packages/syft/proto/core/tensor/share_tensor.proto

```diff
@@ -2,12 +2,12 @@ syntax = "proto3";
 
 package syft.core.tensor;
 
-import "proto/core/tensor/tensor.proto";
 import "proto/lib/numpy/array.proto";
+import "proto/core/common/recursive_serde.proto";
 
 message ShareTensor {
   oneof data {
-    Tensor tensor = 1;
+    syft.core.common.RecursiveSerde tensor = 1;
```

Comment:
Q: This changed from tensor to recursive serde? (also in DP?)

Reply:
It was recursive serde , before itself, Tudor did a serde refactor , which modified it a decorator and change the types. so I had to modify it.